test/spec/var/very-is-raw-json/expect.js

```diff
@@ -0,0 +1,4 @@
+var x = {
+    foo: bar,
+    baz: qwz
+}
```

Comment:
I'd expect there to be a semicolon here

Reply:
ah, that simplifies things then. I wasn't sure if that was required syntax or not ! i'll update it. Node seems to let me not have it:
```
$ node
> var x = {
... foo: 'bar',
... baz: 'qwz'
... }
undefined
> x
{ foo: 'bar', baz: 'qwz' }

```

But if it doesn't hurt to append `;` at the end, that simplifies some logic.